cmd/opentelemetry-collector/main.go

```diff
@@ -46,23 +52,38 @@ func main() {
 	}
 
 	v := viper.New()
+	storageType := os.Getenv(storage.SpanStorageTypeEnvVar)
+	if storageType == "" {
+		storageType = "cassandra"
+	}
+
```

Comment:
Seems like it belongs better in the config file.

Reply:
I would like to keep the parsing of flags and getting the settings from the environment in the main - in one place.